src/Compilers/CSharp/Portable/Compilation/InitializerSemanticModel.cs

```diff
@@ -275,5 +275,10 @@ protected override void AnalyzeBoundNodeNullability(BoundNode boundRoot, Binder
         {
             NullableWalker.AnalyzeWithoutRewrite(Compilation, MemberSymbol, boundRoot, binder, diagnostics, createSnapshots);
         }
+
+        protected override bool IsNullableAnalysisEnabled()
+        {
+            return Compilation.IsNullableAnalysisEnabledIn(Root);
```

Comment:
>return Compilation.IsNullableAnalysisEnabledIn(Root); [](start = 12, length = 53)

This feels inconsistent, I think somewhere else we were including nodes for all field initializers.